src/style.css

```
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block__item {
+  display: flex;
+  flex: 1 0 33.33%;
+  align-items: center;
+  justify-content: center;
+  height: 300px;
+  min-width: 300px;
+}
```

Давай робити відступи між різними групами CSS правил. Так твій буде простіше читати
```suggestion
.block__item {
  display: flex;
  flex: 1 0 33.33%;
  align-items: center;
  justify-content: center;
  
  height: 300px;
  min-width: 300px;
}
```